code/game/machinery/doors/airlock.dm

```diff
@@ -951,7 +951,7 @@ About the new airlock wires panel:
 	else if(!repairing && iscrowbar(C))
 		if(src.p_open && (operating < 0 || (!operating && welded && !src.arePowerSystemsOn() && density && (!src.locked || (stat & BROKEN)))) )
 			playsound(src.loc, 'sound/items/Crowbar.ogg', 100, 1)
-			user.visible_message("[user] removes the electronics from the airlock assembly.", "You start to remove electronics from the airlock assembly.")
+			user.visible_message("[C][user] removes the electronics from the airlock assembly.", "You start to remove electronics from the airlock assembly.")

```

Comment:
Intentional ? [C][user]

Reply:
It's there for testing at the moment. Trying to work out why what I'm doing's not working.